crates/control-interface/src/lib.rs

```diff
@@ -503,6 +503,47 @@ impl Client {
         }
     }
 
+    /// Put a new (or update an existing) label on the given host.
+    ///
+    /// # Errors
+    ///
+    /// Will return an error if there is a communication problem with the host
+    pub async fn put_label(
+        &self,
+        host_id: &str,
+        label: &str,
+        value: &str,
+    ) -> Result<CtlOperationAck> {
+        let subject = broker::put_label(&self.topic_prefix, &self.lattice_prefix, host_id);
+        debug!(%subject, "putting label");
+        let bytes = json_serialize(HostLabel {
+            label: label.to_string(),
+            value: value.to_string(),
+        })?;
+        match self.request_timeout(subject, bytes, self.timeout).await {
+            Ok(msg) => Ok(json_deserialize(&msg.payload)?),
+            Err(e) => Err(format!("Did not receive put label acknowledgement: {e}").into()),
+        }
+    }
+
+    /// Removes a label from the given host.
+    ///
+    /// # Errors
+    ///
+    /// Will return an error if there is a communication problem with the host
+    pub async fn delete_label(&self, host_id: &str, label: &str) -> Result<CtlOperationAck> {
+        let subject = broker::delete_label(&self.topic_prefix, &self.lattice_prefix, host_id);
+        debug!(%subject, "removing label");
```

Comment:
Hey could you add the `host_id` here? Maybe something like `removing label [{}] from host [{}]`?

Reply:
hmnnn I know that as someone debugging something perhaps I'd really want to know *what* the label was removed from, but definitely shouldn't hold up this PR -- we'll cross that :bridge_at_night: if we get there